libsolidity/interface/CompilerStack.h

```diff
@@ -90,7 +90,8 @@ class CompilerStack: boost::noncopyable
 	enum State {
 		Empty,
 		SourcesSet,
-		ParsingPerformed,
+		Parsed,
+		ParsedAndImported,
```

Comment:
Why is it called `ParsedAndImported`? Mostly curious about the `imported` part, since that was not mentioned before.

Reply:
the parse step usually also immediately imports and processes the imported files as well. Now, when you specify "stopAfter=parsing" it will not do the import but just parses what it has directly.

Reply:
The main point behind this feature is to stop before processing the imports and auto-loading other files.